### ISSUE TITLE
redux-oidc-2/index.d.ts (module not found)

### ISSUE THREAD

**Issue by jdavis-software** [ryan-sandy/redux-oidc#1]
@ryan-sandy  running into a import issue, other than that looking at the code it looks good!
```js 
node_modules/redux-oidc-2/index.d.ts' is not a module.
```
Here is the import code
```js
import { CallbackComponent } from "redux-oidc-2";
```
Am I missing something with the configuration of it? I am using CRA just FYI 

Here are the packages as well:

```js
    "oidc-client": "^1.11.5",
    "oidc-client-ts": "^2.2.4",
    "redux-oidc-2": "^1.0.0",
```

**Comment by ryan-sandy** [comment]
I just tested this with a fresh CRA and I'm not duplicating the error. I would try restarting the development server. If that doesn't work, try removing the library and reinstalling.

**Comment by jdavis-software** [comment]
@ryan-sandy yeah I tried both of those last Friday... and again this morning... it has to do with the types in the `tsconfig` 

```json
    "types": [
      "jest",
      "node",
      "cypress",
      "redux-oidc-2"
    ],
```
but that isn't resolving it, take a look at this [stackoverflow ](https://stackoverflow.com/questions/40441252/typescript-typings-give-me-index-d-ts-is-not-a-module), I think we need to add a `npm i --save-dev @types/redux-oidc-2` or something because typescript isn't picking it up as a module. Any thoughts?

**Comment by jdavis-software** [comment]
@ryan-sandy actually looking at the declaration model `index.d.ts` for your package I think the name is `redux-oidc-hooks` not `redux-oidc-2` and is causing it to error

```js
declare module "redux-oidc-hooks" 
```

**Comment by ryan-sandy** [comment]
@jdavis-software I just pushed a new version that should have the correct export. Sorry about that. I missed it when I cloned the repo the first time. If it fixes this issue, let me know and I'll close the issue.

**Comment by jdavis-software** [comment]
@ryan-sandy perfect ty that fixed it but getting this error on the `children` of the `<CallbackComponent>`

![image](https://github.com/ryan-sandy/redux-oidc/assets/21320194/759ed655-6c4f-4424-b87a-355ca40cf0bf)

**Comment by ryan-sandy** [comment]
@jdavis-software Try v1.1.1 now. I added the props to the export in index.ts.

**Comment by M1019171** [comment]
@ryan-sandy  i get the below error now

Only the Authorization Code flow (with PKCE) is supported
Call Stack
 OidcClient.createSigninRequest
  bundle.js:50800:13
 UserManager._signinStart
  bundle.js:51940:48
 UserManager.signinRedirect
  bundle.js:51707:16



---------------------------------------------
my   response_type: "token id_token", in usermamanger

**Comment by jdavis-software** [comment]
@ryan-sandy perfect looks like its working now! @M1019171 make sure your run this before.

```js
npm remove redux-oidc-2
```

then re-install
```js
npm i redux-oidc-2@latest
```

**Comment by ryan-sandy** [comment]
@M1019171 You will also need to install the new userManager

```
npm remove oidc-client
npm install --save oidc-client-ts
```
